libdnf.spec

```diff
@@ -86,7 +86,6 @@ BuildRequires:  pkgconfig(cppunit)
 BuildRequires:  pkgconfig(modulemd-2.0) >= %{libmodulemd_version}
 BuildRequires:  pkgconfig(smartcols)
 BuildRequires:  gettext
-BuildRequires:  gpgme-devel
```

Comment:
We can also remove `gpgme-devel` reference from README.md ([here](https://github.com/rpm-software-management/libdnf/blob/998efa8f41188f9dc4b08b74086c2c3e2a38f73a/README.md?plain=1#L23)).

Reply:
Done